styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: chocolate;
+}
```

Comment:
- [ OPTIONAL ] I noticed that you skipped some of the requirements. You didn't Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config)  as instructed in the project requirements.
- [ ]  You were also to make sure that any files that are not relevant are ignored by git.
- [ ] Kindly note that this was a practice for you to learn how to properly do these things as they will be essential for every other project you undertake at Microverse so even though this is an optional comment, I will suggest that you go through the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) and include all the missing requirements in your project.
- [ ] In future projects, your project will be  invalidated if these requirements aren't met :100: